### PR TITLE
[KV] Wrangler-KV Command Update

### PR DESCRIPTION
**Pull request by Oxyjun**
### Summary

Continuation of the PR from: https://github.com/cloudflare/cloudflare-docs/pull/15985

TLDR of this current PR:

1. Centralising Wrangler-KV commands into `partials/workers/wrangler-commands/kv.mdx`. This is the single source of truth for Wrangler-KV commands.
2. Referencing the KV commands in the "Commands" page in `docs/workers/wrangler/commands.mdx`.
3. Referencing the KV commands in the "Commands" page in `docs/kv/kv-commands.mdx`.
4. Previously, there were some small discrepancies in H4s between the two locations found in (2) and (3). This has been unified. **Specifically, the content from `docs/workers/wrangler/commands.mdx` has been used over the other.**

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Comment by Oxyjun** [comment]
There is currently one pending issue - when the "source of truth" KV commands are rendered in the list of all Wrangler commands, the table of headings on the RHS fails to register duplicated header names. There are several workarounds for this, but these changes need to be properly discussed with PCX.

**Review by thomasgauvin — APPROVED** [review]

**Comment by thomasgauvin** [comment]
> There is currently one pending issue - when the "source of truth" KV commands are rendered in the list of all Wrangler commands, the table of headings on the RHS fails to register duplicated header names. There are several workarounds for this, but these changes need to be properly discussed with PCX.

I think this has been solved, so good to merge?

File: src/content/partials/workers/wrangler-commands/kv.mdx [hunk 1]
```diff
@@ -0,0 +1,498 @@
+---
+{}
+
+---
+
+## `kv namespace`
+
+Manage Workers KV namespaces.
+
+:::note
```
Comment (pedrosousa) [inline src/content/partials/workers/wrangler-commands/kv.mdx:10]: Nit: Ideally we'd avoid having two asides in a row.

File: src/content/partials/workers/wrangler-commands/kv.mdx [hunk 2]
```diff
@@ -0,0 +1,498 @@
...
+
+The `kv ...` commands allow you to manage your Workers KV resources in the Cloudflare network. Learn more about using Workers KV with Wrangler in the [Workers KV guide](/kv/get-started/).
+:::
+
+:::caution
+
+Since version 3.60.0, Wrangler supports the `kv ...` syntax. If you are using versions below 3.60.0, the command follows the `kv:...` syntax. Learn more about the deprecation of the `kv:...` syntax in the [Wrangler commands](/kv/reference/kv-commands/) for KV page.
```
Comment (pedrosousa) [inline src/content/partials/workers/wrangler-commands/kv.mdx]:
```suggestion
Since version 3.60.0, Wrangler supports the `kv ...` syntax. If you are using versions below 3.60.0, the command follows the `kv:...` syntax. Learn more about the deprecation of the `kv:...` syntax in the [Wrangler commands](/kv/reference/kv-commands/#deprecations) for KV page.
```
Maybe we can use the new heading here, since the page is quite long?

File: src/content/partials/workers/wrangler-commands/kv.mdx [hunk 3]
```diff
@@ -0,0 +1,498 @@
...
+
+```sh output
+Deleting the key "my-key" on namespace f7b02e7fc70443149ac906dd81ec1791.
+```
+
+## `kv bulk`
+
+Manage multiple key-value pairs within a Workers KV namespace in batches.
+
+:::note
```
Comment (pedrosousa) [inline src/content/partials/workers/wrangler-commands/kv.mdx:361]: Nit (for later): This note + caution appears multiple times in the page. Could we use yet another partial for this?

File: src/content/partials/workers/wrangler-commands/kv.mdx [hunk 4]
```diff
@@ -0,0 +1,498 @@
...
+```
+
+Refer to the full schema for key-value entries uploaded via the bulk API:
+
+- `key` string required
+  - The key’s name. The name may be 512 bytes maximum. All printable, non-whitespace characters are valid.
+- `value` string required
+  - The UTF-8 encoded string to be stored, up to 25 MB in length.
+- `metadata` object optional
+  - Any arbitrary object (must serialize to JSON) to a maximum of 1024 bytes.
```
Comment (pedrosousa) [inline src/content/partials/workers/wrangler-commands/kv.mdx]:
```suggestion
  - Any arbitrary object (must serialize to JSON) to a maximum of 1,024 bytes.
```

File: src/content/partials/workers/wrangler-commands/kv.mdx [hunk 5]
```diff
@@ -0,0 +1,498 @@
...
+
+Delete a given namespace.
+
+```txt
+wrangler kv namespace delete {--binding=<BINDING>|--namespace-id=<NAMESPACE_ID>} [OPTIONS]
+```
+
+:::caution
+
+This command requires `--binding` or `--namespace-id`.
```
Comment (pedrosousa) [inline src/content/partials/workers/wrangler-commands/kv.mdx:102]: Nit: Currently we write this warning in three different ways along the page:
* This command requires --binding or --namespace-id.
* This command requires a --binding or --namespace-id flag.
* Exactly one of --binding or --namespace-id is required.

**Review by pedrosousa — APPROVED** [review]
